libbeat/tests/system/beat/beat.py

```diff
@@ -58,6 +58,8 @@ def start(self):
                 stderr=subprocess.STDOUT,
                 bufsize=0,
             )
+            # If a "No such file or directory" error points you here, run 
```

Comment:
Good note, it's an error message I wanted to improve for some time.

Reply:
Indeed these are good comments :slightly_smiling_face: 

What about handling these exceptions to print these messages so nobody has to look for these comments here? (It can be done in a future PR in any case)